src/main/kotlin/com/moshbit/katerbase/Utils.kt

```diff
@@ -18,4 +20,10 @@ internal fun ByteArray.sha256(): String {
 
 internal fun String.sha256(): String {
   return this.toByteArray().sha256()
+}
+
+internal fun Date.toIsoString(): String {
+  return SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSX", Locale.GERMANY)
```

Comment:
```suggestion
  return SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSX")
```

Reply:
in this case locale doesn't matter, since we don't format e.g. weekday-name or long-month-name